### PR TITLE
docs: Enhanced Readme

### PR DESCRIPTION
**Pull request by Nesqwik**

### CODE REVIEW

**Review by jonas-martinez — DISMISSED** [review]
Looks good to me, very nice README !

File: README.md [hunk 1]
```diff
@@ -1,12 +1,82 @@
-# Bouncer
+<div id="top"></div>
+<!--
+*** Thanks for checking out the Best-README-Template. If you have a suggestion
+*** that would make this better, please fork the repo and create a pull request
+*** or simply open an issue with the tag "enhancement".
```
Comment (shiipou) [inline README.md:5]: GitHub account without Contributor access to the repository, the creator of the issue cannot set tags on this issue.
Only selecting a template can allow the user to set tags.

**Review by shiipou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Nesqwik) [reply README.md:5]: Thats the comment of the Template i used. I can remove it if it's needed

Comment (shiipou) [inline README.md:5]:
```suggestion
*** or simply open an issue.
```

**Review by shiipou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shiipou) [reply README.md:5]: I think you can just remove the following to be more clear for future contributor :
```suggestion
*** or simply open an issue.
```

**Review by shiipou — APPROVED** [review]

**Review by jonas-martinez — APPROVED** [review]

**Pull request merged**